### DISCUSSION TITLE
ESP-WROOM 32

### DISCUSSION THREAD

**Discussion by FirasKhemir** [risinek/esp32-wifi-penetration-tool#51] [Q&A]
Can I use the ESP-WROOM 32 to use the wifi penetration tool ? or do I need an esp 32 specifically ?

**Comment by risinek** [comment]
As long as the chip is ESP32, then yes ESP-WROOM-32 is perfectly fine.

**Reply by FirasKhemir** [reply]
Thank you.

**Reply by FirasKhemir** [reply]
I managed to flash the esp32 and run an attack but the pcap file is empty and I can't download the hccapx file. any advice you can give ?

**Reply by KevinGaudry** [reply]
same issue here, is there an update on it ? Are we doing somehing wrong ?

**Reply by FirasKhemir** [reply]
@KevinGaudry I kept retrying and eventually I got a file that was not empty, I guess it depends on the signal strength or maybe the state of the access point (if there are devices connected to it or not)

**Reply by mach9977** [reply]
same issue here, i cant catch anything, both files are empty almost all the time. Once i got something, but i can crack it with hashcat, i get result exhausted. Am I doing something wrong?

**Reply by FirasKhemir** [reply]
@mach9977 I tried to crack the only result I got and no luck, it always runs out of time or something, I don't think you're doing something wrong, we just need something better to decode the result.
